src/verilog.l

```diff
@@ -135,16 +135,44 @@ vnum    {vnum1}|{vnum2}|{vnum3}|{vnum4}|{vnum5}
   {ws}                  { FL_FWD; FL_BRK; }  /* otherwise ignore white-space */
   {crnl}                { FL_FWD; FL_BRK; }  /* Count line numbers */
 
+  "clock_enable"        { FL; return yVLT_CLOCK_ENABLE; }
+  "clocker"             { FL; return yVLT_CLOCKER; }
+  "coverage_block_off"  { FL; return yVLT_COVERAGE_BLOCK_OFF; }
   "coverage_off"        { FL; return yVLT_COVERAGE_OFF; }
   "coverage_on"         { FL; return yVLT_COVERAGE_ON; }
+  "full_case"           { FL; return yVLT_FULL_CASE; }
+  "inline"              { FL; return yVLT_INLINE; }
+  "isolate_assignments" { FL; return yVLT_ISOLATE_ASSIGNMENTS; }
   "lint_off"            { FL; return yVLT_LINT_OFF; }
   "lint_on"             { FL; return yVLT_LINT_ON; }
+  "no_clocker"          { FL; return yVLT_NO_CLOCKER; }
+  "no_inline"           { FL; return yVLT_NO_INLINE; }
+  "parallel_case"       { FL; return yVLT_PARALLEL_CASE; }
+  "public"              { FL; return yVLT_PUBLIC; }
+  "public_flat"         { FL; return yVLT_PUBLIC_FLAT; }
+  "public_flat_rd"      { FL; return yVLT_PUBLIC_FLAT_RD; }
+  "public_flat_rw"      { FL; return yVLT_PUBLIC_FLAT_RW; }
+  "public_module"       { FL; return yVLT_PUBLIC_MODULE; }
+  "sc_bv"               { FL; return yVLT_SC_BV; }
+  "sc_clock"            { FL; return yVLT_CLOCK; }
+  "sformat"             { FL; return yVLT_SFORMAT; }
   "tracing_off"         { FL; return yVLT_TRACING_OFF; }
   "tracing_on"          { FL; return yVLT_TRACING_ON; }
 
+  -?"-block"            { FL; return yVLT_D_BLOCK; }
   -?"-file"             { FL; return yVLT_D_FILE; }
+  -?"-function"         { FL; return yVLT_D_FUNCTION; }
   -?"-lines"            { FL; return yVLT_D_LINES; }
+  -?"-module"           { FL; return yVLT_D_MODULE; }
   -?"-msg"              { FL; return yVLT_D_MSG; }
```

Comment:
Thought we were  going to deprecate --msg for --rule?

Reply:
This was running in parallel. I will create a separate PR for the rename and base this on top.